drivers/scheduler/k8s/k8s.go

```diff
@@ -6346,6 +6346,43 @@ func getLabelsFromNodeAffinity(nodeAffSpec *v1.NodeAffinity) map[string]string {
 	return label
 }
 
+// AddNamespaceLabel adds a label key=value on the given namespace
+func (k *K8s) AddNamespaceLabel(namespace string, labelMap map[string]string) error {
+	ns, err := k8sCore.GetNamespace(namespace)
+	if err != nil {
+		return err
+	}
+	ns.SetLabels(labelMap)
```

Comment:
will not override existing labels?

Reply:
Incase same Key name is already present, this will override the value. Otherwise, new key will be appended to the existing Keys.